collections/_priorities/rendering-09-override-the-entire-renderer-using-a-gdextension.md

```diff
@@ -0,0 +1,16 @@
+---
+last_updated: "2024-11-30"
+category: rendering
```

Comment:
After some discussion with others, I think we should remove this. It is something that we have talked about over the years and would like to see. But we all have different ideas on what is possible and what is desirable. I think we need  a clearer idea of the end goal before we include it here